src/emm/cli/git_cli.py

```diff
@@ -0,0 +1,110 @@
+"""CLI interfaces for git actions."""
+
+import click
+import inject
+from rich import print as rprint
+
+from emm.services.git_branch_service import GitBranchService
+from emm.services.validation_service import ValidationService
+
+
+class GitOrchestrator:
+    """Orchestrator for git subcommands."""
+
+    @inject.autoparams()
+    def __init__(self, git_branch_service: GitBranchService) -> None:
+        """
+        Initialize the orchestrator.
+
+        :param git_branch_service: Service to work with git branches.
+        """
+        self.git_branch_service = git_branch_service
+
+    def create_branch(self, branch_name: str, revision: str) -> None:
+        """
+        Create a branch on all repositories.
+
+        :param branch_name: Name of branch to create.
+        :param revision: Revision to base branch off.
+        """
+        modules = self.git_branch_service.create_branch(branch_name, revision)
+        rprint(f"Branch [green]'{branch_name}'[/green] created on: ")
+        for module in modules:
+            rprint(f" - [yellow]{module}[/yellow]")
+
+    def view_branches(self) -> None:
+        """View branches on all enabled repositories."""
+        branches = self.git_branch_service.branch_list()
+        for module in branches:
+            rprint(f"Branches in [yellow]'{module.module_name}'[/yellow]:")
+            for branch in module.output.splitlines():
+                branch_name = branch
+                if branch.startswith("*"):
+                    branch_name = f"[green]{branch}[/green]"
+                rprint(branch_name)
+
+    def switch_branch(self, branch_name: str) -> None:
+        """
+        Switch to the given branch from all repositories.
+
+        :param branch_name: Name of branch to switch to.
+        """
+        modules = self.git_branch_service.switch_branch(branch_name)
+        rprint(f"Switched to [green]'{branch_name}'[/green] in: ")
+        for module in modules:
+            rprint(f" - [yellow]{module}[/yellow]")
+
+    def delete_branch(self, branch_name: str) -> None:
+        """
+        Delete branch from all repositories.
+
+        :param branch_name: Name of branch to delete.
+        """
+        modules = self.git_branch_service.delete_branch(branch_name)
+        rprint(f"Branch [red]'{branch_name}'[/red] delete from: ")
+        for module in modules:
+            rprint(f" - [yellow]{module}[/yellow]")
+
+
+@click.command(context_settings=dict(max_content_width=100))
```

Comment:
Right now, this is just support branch commands. I tried to be consistent with their naming: `git`-`object`-`action`. With this naming scheme related command will show up together in the usage docs.

Reply:
Have you spoken with the people who are currently using our tool? Since we're breaking backwards compatibility, this might be the time to get feedback on how exactly people want to use this and think about this. One option here is to instead further break these actions in sub commands, something like

```
emm git branch-create
emm git branch-update
```
or 
```
emm git branch create
emm git branch update
```

What do you think?

Reply:
I haven't spoken with any user about these changes specifically. In my opinion, the tool is unusable for git operations as-is. When I try to use it locally, I immediately run into limitations. And given that we are still in a beta period, I'm not too worried about breaking backwards compatibility if it improves things.

As far as sub-subcommands go, we could migrate to them. I worry a little about nesting too deeply as each layer we add makes discoverability of the commands worse. But adding layers for evergreen and git commands might work.

Reply:
As discussed in person, I switched over to adding an `evg` subcommand and a `git` subcommand.